sequel/select.js

```diff
@@ -82,7 +82,7 @@ SelectBuilder.prototype.buildSimpleSelect = function buildSimpleSelect(queryObje
     var population = queryObject.instructions[attr].instructions[0];
 
     // Handle hasFK
-    var childAlias = _.find(_.values(self.schema), {tableName: population.child}).identity;
+    var childAlias = _.find(_.values(self.schema), {tableName: population.child}).tableName;
```

Comment:
1. Is this not equivalent to `population.child`?
2. It seems like this _should_ be the identity (alias), since it's used to key the schema object below.  I think the variable name reflects that too.